LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_capture_suppressing_mouse-expected.txt

```diff
@@ -1,7 +1,5 @@
 
 
-Harness Error (TIMEOUT), message = null
-
-TIMEOUT Validate pointer events track pointer movement without pointer capture. Test timed out
-NOTRUN Test pointer capture.
+PASS Validate pointer events track pointer movement without pointer capture.
+FAIL Test pointer capture. assert_equals: Check after hover on capture target: Unexpected events pointerout@target0, pointerleave@target0 expected 0 but got 2
```

Comment:
Looks like these are the same extraneous pointerout/pointerleave events we see in this https://wpt.fyi/results/pointerevents/pointerevent_sequence_at_implicit_release_on_click.html%3Fmouse?label=experimental&label=master&aligned&view=interop&q=label%3Ainterop-2023-events